proposals/restructure-KF-docs-proposal.md

```diff
@@ -0,0 +1,226 @@
+This is an initial proposal, to be iterated on, following the discussion in [#2293](https://github.com/kubeflow/website/issues/2293).
+
+### Kubeflow documentation structure
+
+- **About**
+  - Kubeflow
+  - Use Cases
+  - Contributing to Kubeflow
+  - Community
+  - Google Summer of Code
+  - Docs
+  - Style Guide for Kubeflow docs
+  
+- **Getting Started**
+  - Kubeflow Overview
+  - Installing Kubeflow
+  - Cloud Installation
+    - AWS
+    - Azure
+    - Google Cloud
+    - IBM Cloud
+  - Kubernetes installation
+    - Overview of Deployment on Existing Clusters
+    - Kubeflow Deployment with kfctl_k8s_istio
+    - Multi-user, auth-enabled Kubeflow with kfctl_existing_arrikto
+    - Multi-user, auth-enabled Kubeflow with kfctl_istio_dex
+  - Workstation Installation
+    - Kubeflow on Linux
+    - Kubeflow on macOS
+    - Kubeflow on Windows
+    - MiniKF
+    - Deploy using MiniKF on GCP
+    - Deploying with minikube on a single node
+    - Kubeflow on MicroK8s
+- **Components of Kubeflow**
+  - Central Dashboard
+    - Central Dashboard 
+    - Registration Flow
+  - Metadata
+  - Jupyter Notebooks
+  - Fairing 
+    - Overview of Kubeflow Fairing
+    - Install Kubeflow Fairing
+    - Configure Kubeflow Fairing
+    - Fairing on Azure
+    - Fairing on GCP
+    - Tutorials
+    - Reference
+  - Feature Store
+    - Introduction to Feast
+    - Getting started with Feast
+  - Frameworks for Training
+    - Chainer Training
+    - MPI Training
+    - MXNet Training
+    - PyTorch Training
+    - TensorFlow Training (TFJob)
+  - Hyperparameter Tuning
```

Comment:
I think we can rename it to Katib in proposal, after this PR: https://github.com/kubeflow/website/pull/2312 we will use Katib instead of Hyperparameter Tuning.

WDYT @gaocegege @johnugeorge ?

Reply:
@andreyvelich What would be the adequate title for the component to make the purpose very clear:
1. Katib
2. Katib (HP-tuning)
3. Katib (Auto ML)
4. Katib (Hyperparameter tuning)

Reply:
I think Katib is enough clear.
@gaocegege @johnugeorge Do we want to add AutoML at the name ?

Reply:
Hi @andreyvelich, @gaocegege, @johnugeorge, do you have a decision on this front based on the survey?

Reply:
It seems that most of our users want to leave the Katib Name. Check the results: https://docs.google.com/forms/d/e/1FAIpQLSdbtUn95BfI4iWcgwhvRy38ZG7637dV0hXDAPqx-g_7Nk4Rnw/viewform?usp=sf_link.
We make our final decision after the upcoming AutoML WG meeting on this Wednesday, I will let you know.

/cc @gaocegege @johnugeorge

Reply:
Hi @andreyvelich! Should I switch here to `Katib` then?

Reply:
Yeah, let's keep it with Katib.
In addition, it would be great if you can modify this proposal with updated structure for the Katib docs.
Check it here: https://www.kubeflow.org/docs/components/katib/.
For example: `Using Early Stopping` is missing in this proposal.

Thank you for doing this @RFMVasconcelos!

Reply:
Hi @andreyvelich, thank you for the clarity! 

I will update that to reflect the name and new structure, and resolve this thread. 🚀

Reply:
Done! 🎉